web/src/beta/components/fields/DateTimeField/EditPanel/index.tsx

```diff
@@ -0,0 +1,116 @@
+import Button from "@reearth/beta/components/Button";
+import PanelCommon from "@reearth/beta/components/fields/CameraField/PanelCommon";
+import { useT } from "@reearth/services/i18n";
+import { styled } from "@reearth/services/theme";
+
+import TextInput from "../../common/TextInput";
+import SelectField from "../../SelectField";
+
+import useHooks from "./hooks";
+
+type Props = {
+  onChange?: (value?: string | undefined) => void;
+  onClose: () => void;
+};
+
+const EditPanel: React.FC<Props> = ({ onChange, onClose }) => {
+  const t = useT();
+
+  const {
+    date,
+    time,
+    timezones,
+    selectedTimezone,
+    onDateChange,
+    onTimeChange,
+    getUniqueTimezones,
+    onTimezoneSelect,
+    onDateTimeApply,
+  } = useHooks({ onChange });
+
+  const isButtonDisabled = date.trim() === "" || time.trim() === "";
+  const uniqueTimezones = getUniqueTimezones(timezones);
+
+  return (
+    <PanelCommon title={t("Set Time")} onClose={onClose}>
+      <FieldGroup>
+        <TextWrapper>
+          <Label>{t("Date")}</Label>
+          <TextInput className="customTextInput" type="date" value={date} onChange={onDateChange} />
+        </TextWrapper>
+        <TextWrapper>
+          <Label>{t("Time")}</Label>
+
+          <TextInput className="customTextInput" type="time" value={time} onChange={onTimeChange} />
+        </TextWrapper>
+        <SelectWrapper>
+          <Label>{t("Time Zone")}</Label>
+          <SelectField
+            value={selectedTimezone}
+            className="timezone"
```

Comment:
Is this timezone className doing anything? Since we aren't using any external css files (and you aren't adding any in this PR) that would define a timezone class seems like its maybe a mistake?